### PR TITLE
rev42/tfpdf WIP

### PR DESCRIPTION
**Pull request by timothygwebb**

### CODE REVIEW

File: src/bb-library/HelloWorld.txt [hunk 1]
```diff
@@ -0,0 +1,6 @@
+Anglais : Hello World
```
Comment (fordnox) [inline src/bb-library/HelloWorld.txt:1]: What is the purpose of this file? 
if it is required, it is in wrong location

File: src/bb-library/README.txt [hunk 2]
```diff
@@ -0,0 +1,15 @@
+tFPDF accepts UTF-8 encoded text. It embeds font subsets allowing small PDF files.
```
Comment (fordnox) [inline src/bb-library/README.txt:1]: wrong location of the file, or not needed

File: src/bb-library/ex.php [hunk 3]
```diff
@@ -0,0 +1,26 @@
+<?php
```
Comment (fordnox) [inline src/bb-library/ex.php:1]: wrong location of the file

**Review by fordnox — COMMENTED** [review]
this PR introduces more files than needed.

lots of new font files, that are not needed

**Comment by timothygwebb** [comment]
@fordnox I appreciate the feedback and I will adjust accordingly and mark draft later on today when I get a chance. Do you know exactly what fonts are required for boxbilling. I am at a crossroads of a bug or a lack of understanding because it seems to be wanting more than just DejavuSansCondensed and Bold. I welcome more feedback.

**Comment by rndevuk** [comment]
Maybe just change the fonts to use default ones for now?

**Comment by timothygwebb** [comment]
> Maybe just change the fonts to use default ones for now?

@rndevuk @fordnox What are the default fonts?

**Comment by rndevuk** [comment]
look inside the tfpdf repo theres a folder with a bunch of default fonts

**Comment by fordnox** [comment]
> > Maybe just change the fonts to use default ones for now?
> 
> @rndevuk @fordnox What are the default fonts?

please do the finding yourself. if you are offering PR, i assume you have taken time to understand what are you changing

**Comment by timothygwebb** [comment]
> > > Maybe just change the fonts to use default ones for now?
> > 
> > 
> > @rndevuk @fordnox What are the default fonts?
> 
> please do the finding yourself. if you are offering PR, i assume you have taken time to understand what are you changing

@fordnox  I have done the finding I just like community involvement. It seems here that there has only been four main stream concurrent contributors and I just like to have input and develop as a team since they will be looking over my work as well. I know that there are 4 or 5 default fonts inclusive of Helvetica, times, courier, zing-bats and one other which I am not going to look back and find right now. There is not a need to change the fonts to default it is more work then what I have already put in and will prolong the process.

**Comment by timothygwebb** [comment]
@fordnox I have removed or updated the requested items. This pull request is ready for review.

**Comment by timothygwebb** [comment]
@fordnox I have decided to refactor these at a later date I have had an enlightenment.

**Pull request closed without merging**